tests/Makefile.am

```diff
@@ -4,19 +4,21 @@ if AUTOMAKE_TEST_COMPAT
 TESTS_ENVIRONMENT = $(AM_TESTS_ENVIRONMENT)
 endif
 
-TESTS = referenceFAT12.mkfs              \
-	referenceFAT16.mkfs              \
-	referenceFAT32.mkfs              \
-	referenceFAT32mbr.mkfs           \
-	mkfs-fat32_2_res_sects.mkfs      \
-	check-bad_names.fsck             \
-	check-chain_to_free_cluster.fsck \
-	check-chain_too_long.fsck        \
-	check-chain_to_other_file.fsck   \
-	check-circular_chain.fsck        \
-	check-duplicate_names.fsck       \
-	check-dot_entries.fsck           \
-	check-huge.fsck                  \
+TESTS = referenceFAT12.mkfs                    \
+	referenceFAT16.mkfs                    \
+	referenceFAT32.mkfs                    \
+	referenceFAT32mbr.mkfs                 \
+	mkfs-fat32_2_res_sects.mkfs            \
+	check-backup_boot_sector_mismatch.fsck \
```

Comment:
I would suggest to rename this test file to be shorter, so it would not be needed to realign all other file names. There are lot of other pull requests and changes which may result in merge conflicts just to filename alignment.

What about renaming it to just `check-backup_boot_sec_mism.fsck`?

Reply:
Realign all of file names now already causes merge conflicts and therefore it is not possible to merge this PR yet.